### ISSUE TITLE
PEX6 managed 3'-variant not shown

### ISSUE THREAD

**Issue by MUebe** [Clinical-Genomics/MIP#2062]
Hi!
I noticed that in our case 23289, the 3'UTR polymorphism c.*442_445delTAAA is visible in Scout, but not shown on the front page, although we have the variant saved as a managed variant. This is quite alarming, since we may have been dismissing PEX6 heterozygous variants based on the presumed absence of the polymorphism, when it was in fact not flagged. 
Can we find out why this is happening (differing nomenclature, wrong annotation in managed variant, ...)?
Thanks in advance!
Malin

**Comment by dnil** [comment]
No, sorry, this is another kind of issue. It looks like this one has 60% in gnomAD? 
![Screenshot 2023-10-25 at 10 59 24](https://github.com/Clinical-Genomics/MIP/assets/758570/5328cff0-777f-4790-9b04-6297ede65270)
I'm sure it was called and caught on the frequency hard filter then. 
There is no way something this common could be causative. Are you tracking this for some kind of phenomimic again? We have talked about these kinds of wt alleles before. Ideally they should be handled by an inclusion list directly to MIP, after which we could lower the general hard filter threshold to that of the most common recessive disorder variants.

**Comment by MUebe** [comment]
This specific variant leads to allelic expression imbalance, meaning that a pathogenic variant on the same allele is expressed more strongly than the other allele. The variant itself is not a causative, but in heterozygous form, it can affect the outcome of a pathogenic variant on the same allele.

**Comment by jemten** [comment]
The variant get's called by deepvariant but is filtered by MIP due to the popamax AF being at almost 90%. All variants with a AF above 70% gets filtered currently (we're using gnomad AF, swegen AF and gnomad popmax AF). As you say Daniel, having a whitelist would allow us to save these.

**Comment by ielvers** [comment]
Isn't the point of the managed variant list to act as a whitelist for variants like this? The upstream variant in UGT1A1 is also extremely common but we need to know if it's there, and that one is not filtered out. Have we entered the PEX6 variant wrong in the managed variants list?

**Comment by dnil** [comment]
The managed variants list acts in Scout, loading all variants available. The variant you describe is so common it is filtered away in the pipeline long before it reaches the loading concern of Scout.

**Comment by ielvers** [comment]
This particular variant was the reason the AF cutoff was set to 70% so it would appear in Scout (https://github.com/Clinical-Genomics/MIP/issues/1963). If it's still filtered out, maybe there's another approach that is better for common but clinically important variants?

**Comment by jemten** [comment]
If scout has a whitelist we can use that list to ensure that those variants wouldn't get filtered away

**Comment by jemten** [comment]
Opened a new issue in scout to export the whitelist.

**Comment by dnil** [comment]
Ok, fixed. There is now a command `scout export managed` that will give you a VCF with the managed variants, and a VCF header `fileDate` tag to identify when it was exported. I just realised that the header is a little incomplete, and the sort order not perfect (MT variants will end up between autosomes and sex chromosomes), but please shout if this would cause an issue for you and we can correct.

**Comment by jemten** [comment]
Nice! We will try it out. Aside from som new code blocks in MIP/raredisease, implementing this also requires a new commands in cg to export the whitelist from scout and present it to the pipeline. We will write a little user story around it and prioritise it :)
